browser/plugins/load_graph_on_trigger.plugin.js

```diff
@@ -0,0 +1,44 @@
+(function() {
+
+var LoadGraphOnTrigger = E2.plugins.load_graph_on_trigger = function(core, node) {
+	Plugin.apply(this, arguments)
+
+	this.desc = 'Loads and plays the Vizor file (replacing the current one) on trigger'
+
+	this.input_slots = [{
+		name: 'trigger', 
+		dt: core.datatypes.BOOL,
+		desc: 'When trigger is true, the URL will be loaded in the player',
+		def: null
+	}, {
+		name: 'url',
+		dt: core.datatypes.TEXT,
+		desc: 'Relative URL to published Vizor file, eg. /fthr/moon',
+		def: null
+	}]
+
+	this.output_slots = []
+}
+
+LoadGraphOnTrigger.prototype.update_input = function(slot, data) {
+	Plugin.prototype.update_input.apply(this, arguments)
+
+	if (slot.name === 'trigger' && data && this.inputValues.url) {
```

Comment:
I would do the trigger execution in update_state(), otherwise if both inputs are updated on the same frame, the url is not necessarily up to date when trigger input is updated